README.md

```diff
@@ -29,8 +29,47 @@ The following packages are used internally and are not meant to be used directly
 
 ## Usage
 
+### Basic setup
+
 This example uses React, but the tools are platform agnostic. You should already know the basics of Redux before using this library!
 
+In order to integrate `redux-tools` into your project, you first need to add necessary dependencies:
+
+```
+yarn add @redux-tools/reducers @redux-tools/reducers-react @redux-tools/actions
+```
+
+Next, change your store initialization to the following:
+
+```js
+// initStore.js
+import { createStore, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension/logOnlyInProduction';
+import { makeEnhancer as makeReducersEnhancer } from '@redux-tools/reducers';
+
+const initStore = () => createStore(store => store, composeWithDevTools(makeReducersEnhancer()));
+
+export default initStore;
+```
+
+This will create an empty store.
+
+In order to dynamically inject reducer, we need to specify component (or top of component tree) that uses that reducers data. This component should be connected to redux store like this:
+
+```js
+// YourComponent.js
+import { withReducers } from '@redux-tools/reducers-react';
+import reducer from '../yourReducers/reducerToInject';
+
+// YourComponent definition
+
+export default withReducers({ reducerName: reducer }, { isGlobal: true })(YourComponent);
+```
+
+And that's it! Your application now contains basic redux store with dynamically injected reducer.
```

Comment:
```a basic Redux store with a dynamically...```

also, could you specify that the state structure looks like `state.reducerName` which is managed by this reducer?